src/components/Link/Task.tsx

```diff
@@ -162,9 +166,12 @@ export const LinkTask = () => {
                       hasCheckbox={true}
                       metadata={TaskJson.link}
                       idKey={TaskJson.idKey}
-                      internalChildUrl={`/view/task/`}
                       externalChildUrl={TaskJson.externalUrl}
-                      childTitleAccessor={(e) => e.title}
+                      childTitleAccessor={(e) => (
+                        <ButtonAsLink type="button" onClick={() => onSelectTask(task as unknown as ITask)}>
+                          {e.title}
+                        </ButtonAsLink>
+                      ) as unknown as string}
```

Comment:
We should avoid using `as` where possible to ensure type accuracy. It looks like we're typing a `ReactNode` as a `string` which seems questionable. Also why do we need to type a task to `ITask` via `unknown` when `task` should already be represented as an `ITask` (assuming that's what it is)

Reply:
When I'll refactor this page, this terrible FieldMapping component will be removed, and this type juggling will no longer be necessary.